components/resource_control/src/resource_limiter.rs

```diff
@@ -61,7 +61,7 @@ impl QuotaLimiter {
     }
 
     #[allow(dead_code)]
```

Comment:
cfg(test)

Reply:
The function is used in the following PR